### ISSUE TITLE
Add details in output (html/markdown/console) for changes in schema

### ISSUE THREAD

**Issue by quen2404** [OpenAPITools/openapi-diff#18]
More details should be available for schema changes. 
 - Property path 
 - Changes detail

**Comment by jmini** [comment]
I had a look at the code. As far as can tell, the information is already present in the `ChangedXxxx` objects (`com.qdesrame.openapi.diff.model` package) instanciated by the compare engine. I could browse one `ChangedOpenApi` instance to see exactly what changed at property level.

I guess this issue is about having a nicer description in the report.

**Comment by quen2404** [comment]
Yes exactly ! The information is present in `ChangedXxxx` but not in result currently ! I have to develop the implementation in the `XxxxRenderer` classes.

**Comment by thiagoarrais** [comment]
Is this still useful? I'm interested in working on a PR.

**Comment by cminini** [comment]
Looking forward to this feature. Thanks @thiagoarrais

**Comment by thiagoarrais** [comment]
@cminini: have you taken a look at my PR? I'd really love some feedback.

**Comment by michalchmura** [comment]
This would be greatly appreciated!
